src/index.jsx

```diff
@@ -27,6 +28,7 @@ subscribe(APP_READY, () => {
       <main>
         <Switch>
           <Route path="/orders" component={ConnectedOrderHistoryPage} />
+          <Route path="/receipt" component={ConnectedReceiptPage} />
```

Comment:
I expected the path to be `/receipt/?order_number=:orderId` but it doesn't work that way yet, fix coming soon